Assets/Scripts/Actor Components/Buff.cs

```diff
@@ -44,14 +49,36 @@ public void RemoveBuff()
             if (buffTimeout != null)
             {
                 StopCoroutine(buffTimeout);
+                StopCoroutine(blinkingTime);
```

Comment:
There's the possibility of a null exception here if the Knight uses the buff before it starts blinking. I think you can simply reuse the same coroutine variable for both the bufftimeout and blinking coroutines. Alternatively you can also include checks for the blinkingtime coroutine as well but this might make the code harder to maintain.